src/styles/base.scss

```diff
@@ -34,8 +34,11 @@
     h5,
     h6 {
         a {
-            text-decoration: none;
+            text-decoration: underline;
             color: inherit;
+            &:hover, &:visited {
```

Comment:
IMO we should continue to show the underline even after the link has been visited. We might need to move `color: inherit` into this block, though.

Reply:
Ah, good point. Will do both.